server.js

```diff
@@ -1,10 +1,12 @@
 var serverPort = +process.env.PORT || +process.argv[2] || 80;
+var bindAddress = process.env.BIND_ADDRESS || process.argv[3] || '127.0.0.1';
```

Comment:
That's correct - using the previous version of camp and with the existing server.js I was not able to start the server bound to any interface except '::'.  Binding to '::' causes the server to run on the global IPv6 interface (the IPv6 equivalent of '0.0.0.0') - most servers by default bind to a local-only interface to avoid opening unnecessary external-facing ports for security reasons.  This is somewhat mitigated by the fact that the previous default behaviour bound to an IPv6-only interface, and most systems don't have anything but local-IPv6 support; but binding to 127.0.0.1 is slightly more standardized by way of defaults and provides similar semantics for the vast majority of systems (those that only have loopback IPv6 interfaces).

Additionally, accepting alternate bind addresses via environment variables or command-line arguments increases the flexibility for different deployment scenarios, without the need of a reverse proxy.